### PR TITLE
Add wifi-manager service

### PR DESCRIPTION
**Pull request by SyntaxSugarNet**

### CODE REVIEW

File: stage3/00-install-ap-services/00-packages
Comment (ferenj-gs) [inline stage3/00-install-ap-services/00-packages:1]: Can we please add these as a dependency of the wifi mgr Deb package?

File: stage3/01-install-wifi-manager/01-run.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+on_chroot << EOF
+
+cd /tmp/wifiManager
+
+# Package wifi-manager
+dpkg-buildpackage -us -uc -b
+
+# Install wifi-manager
+apt-get install /tmp/wifi-manager_0.0.1_all.deb
```
Comment (ferenj-gs) [inline stage3/01-install-wifi-manager/01-run.sh:12]: Would it be possible to fire up a local apt repo and just list the package in the list of packages and the. At the end of the build just unconfigure the install apt repo?

**Review by ferenj-gs — COMMENTED** [review]
Very nice job

File: stage3/00-install-ap-services/00-packages
Comment (SyntaxSugarNet) [reply stage3/00-install-ap-services/00-packages:1]: Done in https://github.com/GlobalScope-HUN/EffectiveRange/pull/31/commits/be770c2079449fcf5318708f4c730af8ef8223ac

File: stage3/01-install-wifi-manager/01-run.sh (hunk 1 above)
Comment (SyntaxSugarNet) [reply stage3/01-install-wifi-manager/01-run.sh:12]: Nope, as the native libraries needs to be built here at packaging so sourcing it from a local apt repo would make no sense.

**Pull request merged**